includes/Model/TripFormResponse/TripFormResponse.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Arctic\Model\TripFormResponse;
```

Comment:
Similar suggestion here, I would probably call the namespace `TripForm` and the class `Response`.

Reply:
Good idea. I wasn't sure on the name space since I didn't see adding API access to the other points related to a trip form or an evaluation but I like the consistency. 

I have made that modification.